proto/slashing/slashing.proto

```diff
@@ -25,6 +25,23 @@ service Slasher {
     // Returns if a given beacon block header could be slashable when compared to the slashers history for the proposer.
     // This function is read-only, and does not need the beacon block header to be signed.
     rpc IsSlashableBlockNoUpdate(ethereum.eth.v1alpha1.BeaconBlockHeader) returns (Slashable);
+
```

Comment:
can we separate the endpoint impl to another pr?

Reply:
@shayzluf done